index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div class="main"> 
+    <nav>
+        <label class="my-logo"> My Logo</label>
+    </nav>
+
+    <a href="#" class="my-menu">
+        <div></div>
+        <div></div>
+        <div></div>
+    </a>
+
+    </div>
+    
+    <div class="shape-container">
+        <div class="shape-1"></div>
+        <div class="shape-2"></div>
+    </div>
+    
+
+    <div class="headline">
+        <h2> I’m George
+            Glad to see you!</h2>
+    
+        <p class="primary">I’m a software developer! I can help you
+                 build a product , feature or website Look
+                  through some of my work and 
+                  experience! If you like what you see and
+                   have a project you need coded, don’t 
+                   hestiate to contact me.</p>
+
+        <a href="#"class="connect">LET’S CONNECT </a>
+        
+        <div > 
+            <ul class="socials">
+                <li> <a href="#"><img src="img/Vectortwit.png" class="twit"></a> </li>
+                <li> <a href=""><img src="img/Linkedin icon.png"  alt="linkedin"> </a>  </li>
+                <li> <a href=""><img src="img/Vectorm.png" alt="twiter"></a> </li>
+                <li> <a href=""><img src="img/Vectorch.png" alt="icon"></a> </li>
+
+            </ul>
```

Comment:
- [ ] Good Job for using `ul` tag to group them, and `li` per icon;  for each icon using `a` tag for social icons.
- [ Optional ] Kindly, I recommend  you to add your  social networks with the icons, using `href.`